OpenRA.Mods.Cnc/Traits/Mine.cs

```diff
@@ -59,11 +59,13 @@ bool ICrushable.CrushableBy(Actor self, Actor crusher, BitSet<CrushClass> crushC
 			return info.CrushClasses.Overlaps(crushClasses);
 		}
 
-		bool ICrushable.TryCalculatePlayerBlocking(Actor self, BitSet<CrushClass> crushClasses, out LongBitSet<PlayerBitMask> blocking)
+		LongBitSet<PlayerBitMask> ICrushable.CrushableBy(Actor self, BitSet<CrushClass> crushClasses)
 		{
-			// Fall back to the slow path
-			blocking = default(LongBitSet<PlayerBitMask>);
-			return false;
+			var blocking = info.BlockFriendly ? self.Owner.AlliedPlayerMask : self.World.NoPlayersMask;
```

Comment:
Can we please simplify this down to

```csharp
if (!info.CrushClasses.Overlaps(crushClasses))
	return self.World.AllPlayersMask;

// Friendly units should move around!
return info.BlockFriendly ? self.Owner.AlliedPlayersMask : self.World.NoPlayersMask;
```
(assuming this simplification is correct)?